dev/io.openliberty.jakarta.faces.4.1/build.gradle

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+// Global vars
+File downloadSrcDir = project.file('build/source')
+File downloadJavadocDir = project.file('build/javadoc')
+
+configurations {
+    sourceJar
+    javadocJar
+}
+
+dependencies {
+//    sourceJar 'org.apache.myfaces.core:myfaces-api:4.1.0-RC1:sources'
```

Comment:
Let's remove the comment I see these in artifactory now.

Reply:
I don't see them?